eventlog/service.go

```diff
@@ -119,6 +119,83 @@ func (s *Service) GetEvent(req *GetEventRequest) (*GetEventResponse, error) {
 	}, nil
 }
 
+var days = 24 * time.Hour
+var years = 365 * days
```

Comment:
doesn't seem to be used

Reply:
Well, I just thought it would be useful to know. Like in case I forgot how many days in a year later. But now it's in git so I guess I can nuke it. :)